Landingspagina/Index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>Landingspagina</title>
+</head>
+
+<body>
+
+<header class="outer-container">
+
+    <div class="inner-container">
+
+        <nav>
+            <ul class="Menu">
+                <li>Case Studies</li>
+                <li>Services</li>
+                <li>Contact Us</li>
+            </ul>
+        </nav>
+
+
+        <div class="header-content content-restrictor">
+            <h1>Your website is an investment in the success of your business.</h1>
+            <p id="p-slogan">We are here to help you succeed.</p>
+
+            <a href="#work-together" class="button">Let's work together</a>
+        </div>
+
+        <img class="header-im" src="Assets/squiggle_line_stripe.svg"/>
+    </div>
+
+</header>
```

Comment:
Wellicht dat het qua tijd niet gelukt is, maar de header zou schuin af moeten lopen

Reply:
Ok ik dacht dit een bonus was, dus inderdaad heb ik daar geen aandacht aan besteed. Moet is dat alsnog doen?